src/transaction/boot_cl.c

```diff
@@ -4013,6 +4014,105 @@ boot_define_dual (MOP class_mop)
   return NO_ERROR;
 }
 
+/*
+ * boot_define_db_server :
+ *
+ * returns : NO_ERROR if all OK, ER_ status otherwise
+ *
+ *   class(IN) :
+ */
+static int
+boot_define_db_server (MOP class_mop)
+{
+  SM_TEMPLATE *def;
+  char args_string[64];
+  int error_code = NO_ERROR;
+  const char *index_col_names[2] = { "link_name", NULL };
```

Comment:
This is a similar discussion to whether the system table name should be 'db_server' or 'db_link_server'. Since server information can be used more extensively, how about using 'name' rather than 'link_name'?

Reply:
For now, the name will remain as it is.